Model/Checkout/GetQuote.php

```diff
@@ -295,6 +295,7 @@ public function createQuote(array $orderItems): Quote
      */
     public function getCustomerAddress(DataObject $amwalOrderData, string $refId): AddressInterface
     {
+        $customerAddress = '';
```

Comment:
There is no need to pre-define this since an exception is thrown in case the address resolution fails